std/internal/math/biguintcore.d

```diff
@@ -52,6 +52,130 @@ import std.range.primitives;
 import std.traits;
 
 private:
+
+// compile-time & run-time dipatchers to allows BigInt CTFE for 32 bit OS.
```

Comment:
`/for 32bit OS/d` - it should work for all OS at CTFE?

Reply:
No but now it will likely be the case. Even if CTFE isn't a requirement i think that it's a problem when it works for an arch and not the other